index.js

```diff
@@ -0,0 +1,80 @@
+// 1
+
+const movies = ["The Shawshank Redemption",
+    "The Godfather",
+    "The Dark Knight",
+    "12 Angry Men",
+    "The Lord Of The Rings",
+    "Pulp Fiction",
+    "The Good, the Bad and the Ugly",
+    "Forrest Gump"];
+
+    for (let index = 0; index < movies.length; index++) {
+        const element = movies[index];
+        console.log(element);
+       console.log(getWordCount(element))
+    }
+
+function getWordCount(str) {
+        
+     return str.trim().split(/\s+/).length;
+}
+
+// 2
+
+const body = document.querySelector('body');
+const btn = document.querySelector('.js-btn');
+
+body.insertAdjacentHTML('beforeend', `<button class="js-btn">Click</button>`);
```

Comment:
You add this button **after** you looked for it, meaning you will not find it. Also, `body.appendChild()` is a lot simpler than `body.insertAdjacentHTML()`.